src/components/Home.module.css

```diff
@@ -1,13 +1,9 @@
-body {
-  font-family: 'Open Sans Condensed', sans-serif;
-}
-
 .wrapper {
   display: grid;
   height: 100%;
   grid-template-columns: 1fr 1fr;
-  grid-template-rows: .1fr 1fr .1fr;
-  grid-gap: 1px 1px;
+  grid-template-rows: 1fr 7fr 1fr;
```

Comment:
You can mix units. For example: ` auto 1fr auto` or `auto minmax(1fr, 1024px) auto`.

Basically, you want to describe your content as taking up the remaining space, maybe with a min or max constraint. At the moment you would describe it as seven times larger than the gaps to its sides, which is probably not what you were aiming for.

Also I notice you have three rows but only define two at `grid-template-areas`.